test/test_CellArray.jl

```diff
@@ -53,30 +62,34 @@ mutable struct MyMutableFieldArray{T} <: FieldArray{Tuple{2}, T, 1}
     yxxx::T
 end
 
+const TEST_PRECISIONS = [Float32, Float64]
+for (array_type, Array, CellArray, allowscalar) in zip(array_types, ArrayConstructors, CellArrayConstructors, allowscalar_functions)
```

Comment:
Doesn't it change the output when you move the loop outside of everything?

Reply:
Yeah so now all the tests with different array type and precision are launched on different testsets.

Reply:
Loop restored as it was